packages/flutter_tools/test/general.shard/build_system/targets/web_test.dart

```diff
@@ -310,6 +310,26 @@ void main() {
   }, overrides: <Type, Generator>{
     ProcessManager: () => MockProcessManager(),
   }));
+
+  test('WebReleaseBundle copies all files from web folder', () => testbed.run(() async {
+    fs.file(fs.path.join('web', 'index.html'))
```

Comment:
You should add a file in a directory like web/foo/bar.js to test that it gets copied to the correct path